### DISCUSSION TITLE
Central Registry Node @ Kadaster?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#122] [General]
Het eerste [Key Concept](https://github.com/kadaster-labs/sensrnet-home/blob/master/docs/KeyConcepts.md) in SensRNet is 'Decentralization as given'; het gaat uit van een decentraal systeem. Dat betekent dat een bronhouder, bijvoorbeeld een gemeente, een eigen instantie van de SensRNet Registry Node moet starten om sensoren te kunnen registreren en beheren en onderdeel te worden van het netwerk van Registry Nodes, die gezamenlijk de nationale sensorenregistratie vormen. De architectuur houdt rekening met een grote diversiteit van gebruik - daarom ook 'Decentralization as given' - en dat levert ook grote flexibiliteit in [netwerk ontwerp](https://github.com/kadaster-labs/sensrnet-home/blob/master/docs/Network.md) op: heel Registry Nodes of slechts een beperkt aantal.

Zou het - zeker voor het begin, maar misschien wel blijvend - een idee zijn om _ook_ een centrale Registry Node beschikbaar te maken voor de bronhouders die zelf geen Registry Node willen / kunnen beheren?

Voordelen: Geen beheer, lage kosten (gedeelde kosten met andere bronhouders die daar gebruik van maken)

Nadelen: Geen integratie met eigen systemen mogelijk en beschikbaarheid is beperkt tot 99% (SLA nog nader vast te stellen, maar als idee; in ieder geval is beschikbaarheid niet in eigen beheer)

Wat zijn overwegingen in deze?

**Comment by marcvanandel** [comment]
@mario-schrijver, is dit ook wat jij in gedachten had voor een kleine gemeente die mogelijk wil aanhaken?

**Comment by marcvanandel** [comment]
Overigens ben ik hier geen voorstander van. Het is _tegen_ de key concepten van SensRNet en die zijn met een reden vóór een decentrale opzet. Een Central Registry Node is juist een tegenbeweging om toch een centraal systeem te zijn 😨

**Comment by mikegroels** [comment]
Is die "kleine gemeente die mogelijk wil aanhaken" al in beeld?
We hebben het er wel over dat zij dit misschien niet zouden kunnen (betalen), maar zeker weten doen we het niet.

Eventueel zou het ook een optie zijn dat ze gebruik maken van een regio-gemeente of provinciehoofdstad. Dat was ook een van de scenario's die ik ooit voorbij heb zien komen (1 node per provincie of iets dergelijks). De integratie met eigen systemen zou geen probleem moeten zijn, wel een iets grotere uitdaging, maar dat is dan een afweging.

**Reply by marcvanandel** [reply]
Yup. Dat is zeker een mogelijkheid!! Deze staat ook als mogelijke optie bij de '[netwerk opties](https://kadaster-labs.github.io/sensrnet-home/Network/)' 😉 Dit is vooral afhankelijk van de partijen die aanhaken bij SensRNet en hoe het netwerk gaat groeien...
